src/main/java/se/liu/ida/hefquin/cli/RunBGPOverNeo4j.java

```diff
@@ -102,11 +88,7 @@ protected String getSummary() {
 	protected void exec() {
 		final BGP bgp = getBGP();
 
-		final NodeMapping nodeMapping = new NodeMappingToURIsImpl(NSNODE);
-		final NodeLabelMapping nodeLabelMapping = new NodeLabelMappingToURIsImpl(NSNODELABEL);
-		final EdgeLabelMapping edgeLabelMapping = new EdgeLabelMappingToURIsImpl(NSRELATIONSHIP);
-		final PropertyNameMapping propertyNameMapping = new PropertyNameMappingToURIsImpl(NSPROPERTY);
-		final LPG2RDFConfiguration conf = new LPG2RDFConfigurationImpl(NodeFactory.createURI(LABEL), nodeMapping, nodeLabelMapping,edgeLabelMapping,propertyNameMapping);
+		final LPG2RDFConfiguration conf = modLPG2RDFConfiguration.getLPG2RDFConfigurationCatalog();
 
 		final Pair<CypherQuery, Map<CypherVar,Var>> tRes = performQueryTranslation(bgp, conf);
```

Comment:
By your current implementation of `ModLPG2RDFConfiguration`, the `getLPG2RDFConfigurationCatalog()` function may return `null`, which will result in an exception when calling `performQueryTranslation` with it. To avoid this issue, `getLPG2RDFConfigurationCatalog()` should never return `null`. Instead, if the command-line argument is not provided, then a default LPG-to-RDF configuration should be created (for instance, by using the constants `NSNODE`, `NSNODELABEL`, etc. as were here in this class before).